manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/workers/resources/ProcessorWorker.java

```diff
@@ -0,0 +1,74 @@
+package com.redhat.service.smartevents.manager.v2.workers.resources;
+
+import java.util.concurrent.Callable;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.infra.v2.api.models.DefaultConditions;
+import com.redhat.service.smartevents.manager.core.workers.Work;
+import com.redhat.service.smartevents.manager.v2.persistence.dao.ManagedResourceV2DAO;
+import com.redhat.service.smartevents.manager.v2.persistence.dao.ProcessorDAO;
+import com.redhat.service.smartevents.manager.v2.persistence.models.Processor;
+
+@ApplicationScoped
+public class ProcessorWorker extends AbstractWorker<Processor> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProcessorWorker.class);
+
+    private static final String PROCESSOR_WORKER_CLASSNAME = Processor.class.getName();
```

Comment:
One nitpick - the `PROCESSOR_WORKER_CLASSNAME` name makes me expect its value to be `ProcessorWorker.class.getName()`. 

I have checked it is actually consistent with `BridgeWorker.BRIDGE_WORKER_CLASSNAME` and it also makes sense this way, it is just not obvious. I am totally fine with keeping it as it is, just maybe a little food for thought for future refactoring?

Wdyt?

Reply:
Hi @winklerm. The `Worker` acts on certain `ManagedResources`.

We find a `Worker` that can handle `Work` for a `ManagedResource` [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/workers/quartz/WorkJob.java#L37). The `Work` is constructed from the `ManagedResource` [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/workers/quartz/QuartzWorkManagerImpl.java#L59) using [this](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/manager-parent/manager-core/src/main/java/com/redhat/service/smartevents/manager/core/workers/Work.java#L50). So.. the class name of the `ManagedResource` is used in the `accept` method on the `Worker` that uses the constant here.

Does this make more sense?

I however agree with your comment and will change both in due course (once this PR is merged).

Reply:
Yes, it totally makes sense, thanks! 

My only concern (and indeed just a nitpick) was that the name makes you expect something slightly different from what is stored there (which is probably more apparent when someone not familiar with the code is reading it). But overall, it definitely makes sense, so it is just about improving further the readability.